nix/17_09.nix

```diff
@@ -3,7 +3,6 @@ let
 
 in
   fetchNixpkgs {
-     rev          = "3389f23412877913b9d22a58dfb241684653d7e9";
-     sha256       = "1zf05a90d29bpl7j56y20r3kmrl4xkvg7gsfi55n6bb2r0xp2ma5";
-     outputSha256 = "0wgm7sk9fca38a50hrsqwz6q79z35gqgb9nw80xz7pfdr4jy9pf8";
+     rev          = "e1ad1a0aa2ce6f9fd951d18181ba850ca8e74133";
+     sha256       = "0vk5sjmbq52xfrinrhvqry53asl6ppwbly1l7ymj8g0j4pkjf7p1";
```

Comment:
This rev is just what I'm using elsewhere and represents nixpkgs 19.03

I realize that this is a pretty big jump and that it might need to change.

Reply:
I also realize that this file is called `17_09.nix`, so that would also need to change.